src/color-picker/color-picker.js

```diff
@@ -80,6 +80,10 @@ const Self = class ColorPicker extends NudeElement {
 					this._el.sliders.insertAdjacentHTML("beforeend", `<channel-slider space="${ this.space.id }" channel="${ channel }"></channel-slider>`);
 				}
 			}
+
+			if (change.oldInternalValue || change.oldAttributeValue) {
+				this.render();
```

Comment:
Why? A comment would be helpful here!

Reply:
When we first come here on component initialization, there is no need to call `render()` since it'll be rendered automatically after it's connected. So, we only need to re-render the component when `space` actually changes _after_ the component is mounted.

There were no issues when I called `render()` without any condition, but I thought I could optimize it a bit by not re-rendering the component when there is no need.